configs/metro.json

```diff
@@ -0,0 +1,27 @@
+{
+  "index_name": "jest",
+  "start_urls": [
+    {
+      "url": "http://facebook.github.io/metro-bundler/docs/",
```

Comment:
They plan to change this

Reply:
good call, let's use the one that's online first